src/Symfony/Component/HtmlSanitizer/Parser/MastermindsParser.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HtmlSanitizer\Parser;
+
+use Masterminds\HTML5;
+use Symfony\Component\HtmlSanitizer\Parser\Exception\ParsingFailedException;
+
+/**
+ * @author Titouan Galopin <galopintitouan@gmail.com>
+ *
+ * @final
+ */
+class MastermindsParser implements ParserInterface
+{
+    public function parse(string $html): \DOMNode
```

Comment:
Inheritdoc? Or is this not needed?

Reply:
Only necessary when there is an additional comment.